lib/nitcorn/file_server.nit

```diff
@@ -124,6 +130,20 @@ class FileServer
 		return response
 	end
 
+	# Answer the `default_file` if any.
+	fun answer_default: HttpResponse do
+		var default_file = self.default_file
+		if default_file == null then
+			return new HttpResponse(404)
+		end
+
+		var local_file = (root / default_file).simplify_path
+		if not local_file.file_exists then
+			return new HttpResponse(404)
+		end
+		return answer_file(local_file)
```

Comment:
`answer_file` does already the file does not exist 404 check. No need to do it here again